modules/core/shared/src/main/scala/io/circe/Error.scala

```diff
@@ -45,6 +45,11 @@ object ParsingFailure {
  * decoding history resulting in the failure.
  */
 sealed abstract class DecodingFailure(val reason: Reason) extends Error {
+  // Added to satisfy MiMA
+  def this(message: String) = {
```

Comment:
Do you want to deprecate this constructor?

Reply:
If it's just for MiMa, you can make it `private[DecodingFailure]`, but if it was there before, it would be a source-breaking change.  Either way, deprecation sounds smart.